bulkimport_example_test.go

```diff
@@ -0,0 +1,112 @@
+package mssql_test
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+
+	mssql "github.com/denisenkom/go-mssqldb"
+)
+
+const (
+	createTestTable = `CREATE TABLE test_table(
+		[id] [int] IDENTITY(1,1) NOT NULL,
+		[test_nvarchar] [nvarchar](50) NULL,
+		[test_varchar] [varchar](50) NULL,
+		[test_float] [float] NULL,
+		[test_datetime2_3] [datetime2](3) NULL,
+		[test_bitn] [bit] NULL,
+		[test_bigint] [bigint] NOT NULL,
+		[test_geom] [geometry] NULL,
+	CONSTRAINT [PK_table_test_id] PRIMARY KEY CLUSTERED
+	(
+		[id] ASC
+	) ON [PRIMARY]);`
+	dropTestTable = "IF OBJECT_ID('test_table', 'U') IS NOT NULL DROP TABLE test_table;"
+)
+
+// This example shows how to perform bulk imports
+func ExampleCopyIn() {
+	flag.Parse()
+
+	if *debug {
+		fmt.Printf(" password:%s\n", *password)
+		fmt.Printf(" port:%d\n", *port)
+		fmt.Printf(" server:%s\n", *server)
+		fmt.Printf(" user:%s\n", *user)
+	}
+
+	connString := makeConnURL().String()
+	if *debug {
+		fmt.Printf(" connString:%s\n", connString)
+	}
+
+	conn, err := sql.Open("sqlserver", connString)

```

Comment:
Don't call this a "conn". Use db or pool please. This causes alot of confusion, because in ADO this would often be a single connection, but here we don't want people to open a new pool over and over again.